### DISCUSSION TITLE
Open Assignment hatası

### DISCUSSION THREAD

**Discussion by nuricanacar** [19/classroom#16] [Anonim] [answered]
Clasroomu doğru bir şekilde kurdum. Ödevi alırken "Open in Visual Studio Code" kısmına bastığımda fotoğraftaki sorunla karşılaşıyorum. VS Code'de WSL:Clasroom kısmında açtığımda ise her hangi bir sorun gelmeden Vs code tepkisiz kalıyor. 
![Ekran Görüntüsü (2)](https://user-images.githubusercontent.com/91951656/157936780-f9fdb725-9dbc-438a-9c2e-e205ce0cf58f.png)

GitHub üzerinden düzenleme yaptığımda klasörler geliyor. "assignment.c" dosyasına kodları yazdığımda lets grad yapamıyorum. Vs Code de Klasörü Aç kısmında her hangi bir "helloworld-nuricanacar" isimli dosya bulamıyorum. 
![Ekran Görüntüsü (3)](https://user-images.githubusercontent.com/91951656/157938219-2b8b6c12-7315-413f-b206-7e381a856273.png)
Bu kısıma "Open Remote Repository" kısmından erişim sağlıyorum.

Sorunu nasıl çözebilirim veya nerede yanlış yapıyorum ?

**Comment by nuricanacar** [comment, marked as answer]
Sorunu bir kaç arkadaşın yardımıyla çözmüş bulunmaktayım.

İlk olarak fotoğraftaki linki kopyalıyoruz.
![1](https://user-images.githubusercontent.com/91951656/158012916-21b2f08d-82de-4a5b-8c44-449a8c16a343.png)

Sonra VS Code uygulamasını açarak Ctrl+Shift+P Tuşlarına basıyoruz ve çıkan yere "Git klonla" yazıyoruz.Kutu içindekine basıyoruz.
![2](https://user-images.githubusercontent.com/91951656/158013021-f4115872-f02f-4640-b4f4-5b743015a776.png)

Bastıktan sonra çıkan kısıma kopyaladığımız linki yapıştırıyoruz ve "URL'den klonla" kısmına basıyoruz. Sonra gelen "Klasör Seç" penceresinde "Depo Konumunu Seçin" butonuna basıyoruz.
![12](https://user-images.githubusercontent.com/91951656/158013235-787d10cf-2407-4452-9a9b-af39c60bf1ad.png)

Ve sorun çözüldü klonlama işlemi tamam.

**Comment by roktas** [comment]
Sorunu kendi başına çözmen güzel ama VS Code linkinin çalışması lazımdı.  Muhtemelen Github Classroom eklentisinde (belki geçici) bir sorun var.

**Comment by roktas** [comment]
Sorununa biraz daha yakında baktım.  Tahminim şu kullanıcı adın (`Nuric Can Acar`) ve dolayısıyla ev klasörün (`C:\Users\Nuri Can Acar`) boşluk karakteri içerdiğinden Git klonlama yaparken boşluklardan dolayı bir değil 3 ayrı string görüyor.  Özetle, kullanıcı adını boşluk kullanmayacak şekilde ayarlarsan bu sorun (muhtemelen) olmayacaktır.  Buna benzer (aynı değil) bir konuya kurulum dokümanlarınde değindim: https://classroom.alaturka.dev/intro/install.html#k%C4%B1s%C4%B1tlamalar

**Reply by nuricanacar** [reply]
Teşekkür ederim hocam
